scripts/abundance-dist.py

```diff
@@ -95,6 +98,10 @@ def main():
         sys.exit(1)
     hash_fp = open(args.output_histogram_filename, 'w')
 
+    if args.Header_names:
+        print("k-mer_abundance", "K-mer_count", "Cumulative_count" , 
```

Comment:
The same print function is used already in the same script !
https://github.com/ged-lab/khmer/blob/master/scripts/abundance-dist.py#L106